begin update for 2.7.0 (#7172)
rancher/dashboard: shell/content/docs/en-us/whats-new.md

---
title: What's New in 2.7
---

## New in 2.7.0

### Rancher Extensions

- You can now bring your own apps' UI and integrate it with the Rancher Dashboard to leverage a single UI. This relies on feature charts that would be manageable via a catalog. 

> 

Reviewer: @leflambeur Should we add something in here for support for OCI for helm for fleet?

Author: @deniseschannon I've grabbed top-level blurbs from a release notes draft, so the 2.7.0 material has more meat. At this point, I've not received much else from other teams but based on what's going into this version, this may be all.

---
title: What's New in 2.6
---

## New in 2.6.9

> - Alerts within the UI regarding network connectivity are now off by default with a toggle to enable them. This is to address usability uncovered as part of disconnected web sockets.

Reviewer 2: The second sentence is a bit confusing to me, I would say more like: 
```suggestion
- Alerts within the UI regarding network connectivity are now off by default with a toggle to enable them. This will address usability concerns when web sockets are disconnected.
```

---
title: What's New in 2.7
---

## New in 2.7.0

### Rancher Extensions

> - You can now connect other apps to the Rancher Manager Dashboard to leverage a single UI. This relies on feature charts that would be manageable via a catalog.

Reviewer: I'm not sure how exactly to elaborate on this, but this doesn't quite feel right for the product. 

It doesn't give indication that this would allow you to bring in your own UI for your own app to be integrated into the Rancher Manager dashboard.

Also note, that Rancher Manager is still not an external facing term so we should be careful of how and where we use these words.

Author: I reworded it with your feedback, though it's agreed that we need someone to refine this.
